ova/Libraries/provision-opendistro.sh

```diff
@@ -0,0 +1,278 @@
+   echo -e "\t-h   | --help Shows help"
+   exit $1 # Exit script after printing help
+}
+
+
+## Install the required packages for the installation
+installPrerequisites() {
+    logger "Installing all necessary utilities for the installation..."
+    $sys_type install curl unzip wget libcap -y -q 
+    echo -e '[AdoptOpenJDK] \nname=AdoptOpenJDK \nbaseurl=http://adoptopenjdk.jfrog.io/adoptopenjdk/rpm/centos/$releasever/$basearch\nenabled=1\ngpgcheck=1\ngpgkey=https://adoptopenjdk.jfrog.io/adoptopenjdk/api/gpg/key/public' | tee /etc/yum.repos.d/adoptopenjdk.repo $debug
+    $sys_type install adoptopenjdk-11-hotspot -y -q 
+    export JAVA_HOME=/usr/
+
+    if [ "$?" != 0 ]; then
+        echo "Error: Prerequisites could not be installed"
+        exit 1;
+    else
+        logger "Done"
+    fi
+}
+
+## Add the Wazuh repository
+addWazuhrepo() {
+    WAZUH_MAJOR="$(echo ${WAZUH_VERSION} | head -c 1)"
+    logger "Adding the Wazuh repository..."
+    rpm --import https://packages.wazuh.com/key/GPG-KEY-WAZUH 
+    if [ "${STATUS_PACKAGES}" = "prod" ]; then
+      logger "Adding production repository..."
+      echo -e "[wazuh_repo]\ngpgcheck=1\ngpgkey=https://packages.wazuh.com/key/GPG-KEY-WAZUH\nenabled=1\nname=EL-$releasever - Wazuh\nbaseurl=https://packages.wazuh.com/${WAZUH_MAJOR}.x/yum/\nprotect=1" | tee /etc/yum.repos.d/wazuh.repo $debug
+    elif [ "${STATUS_PACKAGES}" = "dev" ]; then
+      logger "Adding development repository..."
+      echo -e '[wazuh_pre-release]\ngpgcheck=1\ngpgkey=https://packages-dev.wazuh.com/key/GPG-KEY-WAZUH\nenabled=1\nname=EL-$releasever - Wazuh\nbaseurl=https://packages-dev.wazuh.com/pre-release/yum/\nprotect=1' | tee /etc/yum.repos.d/wazuh.repo $debug
+    fi
+
+    if [ "$?" != 0 ]; then
+        echo "Error: Wazuh repository could not be added"
+        exit 1;
+    else
+        logger "Done"
+    fi
+}
+
+## Wazuh manager
+installWazuh() {
+
+    logger "Installing the Wazuh manager..."
+    $sys_type install wazuh-manager-${WAZUH_VERSION} -y -q 
+    if [ "$?" != 0 ]; then
+        echo "Error: Wazuh installation failed"
+        exit 1;
+    else
+        logger "Done"
+    fi
+}
+
+## Elasticsearch
+installElasticsearch() {
+
+    logger "Installing Open Distro for Elasticsearch..."
+    $sys_type install opendistroforelasticsearch-${OPENDISTRO_VERSION} -y -q 
+
+    if [ "$?" != 0 ]; then
+        echo "Error: Elasticsearch installation failed"
+        exit 1;
+    else
+        logger "Done"
+
+        logger "Configuring Elasticsearch..."
+
+
+        curl -so /etc/elasticsearch/elasticsearch.yml https://raw.githubusercontent.com/wazuh/wazuh-documentation/${BRANCH}/resources/open-distro/elasticsearch/7.x/elasticsearch_all_in_one.yml --max-time 300
+        
+        curl -so /usr/share/elasticsearch/plugins/opendistro_security/securityconfig/roles.yml https://raw.githubusercontent.com/wazuh/wazuh-documentation/${BRANCH}/resources/open-distro/elasticsearch/roles/roles.yml --max-time 300
```

Comment:
This URL `https://raw.githubusercontent.com/wazuh/wazuh-documentation/${BRANCH}` is repeated through the script several times. I recommend you to create a variable to store this URL and then, run:
```shellsession
curl -so ${VAR_NAME}/resources/open-distro....
```

Reply:
Done in 53208f3d9e390ee69e4f23799af5b3009001d9c2